lib/utils/constants_file.dart

```diff
@@ -1,8 +1,9 @@
 abstract class Constants {
-  static const double imageHeight = 150;
-  static const int modNumber = 6;
   static const double fontSizeTitle = 25;
-  static const double fontSizeText = 40;
+  static const double fontSizeText = 30;
   static const int numberOfDice = 4;
-  static const double paddingText = 70;
+  static const double paddingImage = 20;
+  static const double paddingBottomNav = 40;
+  static const double paddingHorizontal = 20;
+  static const double paddingVertical = 10;
 }
```

Comment:
You can remove the "_file" from the name of the file, with the first part it's enough

Reply:
Fixed